### PR TITLE
Fasih/fix: fix console error

### PR DESCRIPTION
**Pull request by fasihali-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Comment by mohsen-deriv** [comment]
please provide updates on this PR, and if it's not needed please close it, thanks :pray:

File: src/pages/reset-password/index.tsx [hunk 1]
```diff
@@ -35,19 +35,8 @@ const StyledButton = styled(Button)`
...
-    if (email_error) {
-        errors.email = email_error
-    }
-
-    return errors
-}
-
 const ResetPassword = () => {
+    const [apiError, setApiError] = useState<string | null>('')
     const initialValues: EmailType = { email: '' }
```
Comment (aswathy-deriv) [inline src/pages/reset-password/index.tsx:40]: If i am not wrong,do we need this const `initialValues`?

**Review by hasan-deriv — APPROVED** [review]

File: src/pages/reset-password/index.tsx (hunk 1 above)
Comment (fasihali-deriv) [reply src/pages/reset-password/index.tsx:40]: Hi @aswathy-deriv thanks for highlighting, but we are using this on line number `109`. 🙏

**Review by Nuzhy-Deriv — APPROVED** [review]

**Review by aswathy-deriv — APPROVED** [review]

**Review by eduard-deriv — APPROVED** [review]

**Comment by eduard-deriv** [comment]
lgtm

File: src/pages/reset-password/index.tsx [hunk 2]
```diff
@@ -133,7 +140,7 @@ const ResetPassword = () => {
                                 />
                             </InputGroup>
                             <Text align="center" color="red">
-                                {status.error}
+                                {apiError ? apiError : status.error}
```
Comment (mayuran-deriv) [inline src/pages/reset-password/index.tsx:143]:
```suggestion
                                {apiError || status.error}
```

**Review by NikitK-deriv — APPROVED** [review]

File: src/pages/reset-password/index.tsx (hunk 2 above)
Comment (fasihali-deriv) [reply src/pages/reset-password/index.tsx:143]: Hi @mayuran-deriv thanks for pointing this. 🙏 
Is there any performance difference between both techniques ?

Comment (mayuran-deriv) [reply src/pages/reset-password/index.tsx:143]: hi, @fasihali-deriv 
there shouldn't be a significant difference between the two expressions. but apiError || status.error expression is shorter and more idiomatic, which can enhance code readability.

**Review by mayuran-deriv — APPROVED** [review]

**Review by meenakshi-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**